src/applications/disability-benefits/686c-674/README.md

```diff
@@ -0,0 +1,71 @@
+App Name: `Form 21-686c`
+Active engineers: Micah Chiang (front end), Jesse Cohn (front end), Kathleen Crawford (back end), Kevin Musiorski (back end)
+Form ID (if different from app name): `21-686c with 21P-527EZ and 21-674`
+
+# Background
+From time to time Veterans need to make updates to the dependents they get benefits for. These updates can be when a divorce happens, or a death, or a new child is born, etc. The 686c form is how these changes to dependents are made.
+
+## How the form works for the user
+
+Since the Veteran may be making any one of various changes to dependents, there are multiple paths, or workflows, they can take through the form. When the veteran starts the form they are shown an introduction page with some information about what is in the form. Then they are taken to a "wizard" page where they can choose what workflow they want to take though the form. The paths the Veteran can take through the form can be any of the following -
+
+- `Add your spouse`
+- `Add a spouse`
+- `Remove a divorced spouse`
+- `Remove a spouse, child or dependent parent who has died`
+- `Remove a stepchild who has left your household`
+- `Remove a child 18 to 23 years old who has stopped attending school`
+- `Add a child 18 to 23 years old</strong> who'll be attending school` (VA Form 21-674)
+
+> You will notice that the last workflow is technically a totally separate form (the form 21-674). The stakeholders wanted this form added as a workflow to this form since it deals with making changes to a dependent. That workflow actually submits using a separate job than the 686c does. This is detailed further in the back end section.
+
+The Veteran can choose as many or as few of these workflows at a time. This means the Veteran could potentially come into the form and select multiple workflows and we show them only the parts they need to see. For instance, if you choose to `Add your spouse` AND choose `Remove a divorced spouse` we would show you the pages dealing with those two workflows but ONLY those two workflows. This is done using `depends` in the code and is detailed below.
```

Comment:
It could be worth dropping a link to the `depends` [documentation](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/common-patterns-for-building-forms#conditionally-including-a-page) and maybe an example in the 686c form config.

Reply:
Great idea! Adding this.